### PR TITLE
refactor(react-ui-core): change drawer's animation to use translateY

### PR DESCRIPTION
**Pull request by nray621**
affects: @rentpath/react-ui-core

### CODE REVIEW

File: packages/react-ui-core/src/Drawer/Drawer.js [hunk 1]
```diff
@@ -72,42 +71,33 @@ export default class Drawer extends Component {
...
-                    height: `${height - x}px`,
-                  }}
-                >
-                  {children}
-                </div>}
-              </div>
-            )
-          }
-        </Motion>
+        <div>
```
Comment (mneumark) [inline packages/react-ui-core/src/Drawer/Drawer.js]: What is the reason for this div?

File: packages/react-ui-core/src/Drawer/Drawer.js [hunk 2]
```diff
@@ -72,42 +71,33 @@ export default class Drawer extends Component {
 
     return (
       <div
-        className={classnames(theme.Drawer, className)}
+        className={classnames(
+          theme.Drawer,
+          theme[this.state.visible ? 'Drawer-on' : 'Drawer-off'],
```
Comment (mneumark) [inline packages/react-ui-core/src/Drawer/Drawer.js]: You seem to use this.state.visible a lot. Maybe pull it out above?

Comment (nray621) [reply packages/react-ui-core/src/Drawer/Drawer.js]: yup. good call

**Comment by nray621** [comment]
I don't know that it is react motion being unusable, but it seemed to be having trouble animating heavier things like the listing carousel. To be honest I haven't really looked into why exactly this is smoother, but I just changed it to animate the same way rent-js animates their listing carousel.

**Pull request merged**